frontend/app/globals.css

```
@@ -2,8 +2,8 @@
 @tailwind components;
 @tailwind utilities;
 
-@import "highlight.js/styles/github-dark.css";
-/* or any other theme you prefer */
+/* Light mode: github light theme. Dark mode tokens are overridden below. */
+@import "highlight.js/styles/github.css";
```

Stylelint (17.10.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)